crates/turborepo-lib/src/config/user.rs

```diff
@@ -144,19 +145,36 @@ mod test {
         Ok(())
     }
 
+    struct TestCase {
+        config_key: &'static str,
+        config_value: &'static str,
+        env_var_key: &'static str,
+        env_var_value: &'static str,
+        expected: Option<&'static str>,
+    }
+
+    static TOKEN_ENV_VARS: [&'static str; 2] = ["TURBO_TOKEN", "VERCEL_ARTIFACTS_TOKEN"];
+
     #[test]
     fn test_env_var_trumps_disk() -> Result<()> {
         let mut config_file = NamedTempFile::new()?;
         writeln!(&mut config_file, "{{\"token\": \"foo\"}}")?;
-        let env = {
-            let mut map = HashMap::new();
-            map.insert("TURBO_TOKEN".into(), "bar".into());
-            map
-        };
-        let config = UserConfigLoader::new(config_file.path().to_path_buf())
-            .with_environment(Some(env))
-            .load()?;
-        assert_eq!(config.token(), Some("bar"));
+
+        for (idx, env_var) in TOKEN_ENV_VARS.into_iter().enumerate() {
+            let env_var_value = format!("bar{}", idx);
+
+            let env = {
+                let mut map = HashMap::new();
+                map.insert(env_var.into(), env_var_value.clone());
+                map
+            };
+            let config = UserConfigLoader::new(config_file.path().to_path_buf())
+                .with_environment(Some(env))
+                .load()?;
+
+            assert_eq!(config.token(), Some(&env_var_value));
+        }
```

Comment:
Would be helpful to have a test showing which env var takes precedence. From what I understand this tests that either one works. I was just trying to answer what happens when both are set.